source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,266 @@
+
+Search Text by a Term
+~~~~~~~~~~~~~~~~~~~~~
+
+Pass the term as a string to the ``Filters.text()`` method to search for
+a term in your text search. 
+
+If you want to match multiple terms in your text index, separate your
+terms with spaces. The ``Filters.text()`` method performs a logical OR on
+each term and returns documents that contain any of the terms. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "Fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin caseSensitiveExample
+   :end-before: end caseSensitiveExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] }
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass the phrase with escaped quotes to the ``Filters.text()`` method to
+search for a phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+phrase "Fate of the Furious". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Prefix each term you want to exclude in your text search with a minus
+sign ``-``. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "Furious", but do not contain the term "Fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin negateExample
+   :end-before: end negateExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+.. _sort_by_relevance:
+
+Sort by Relevance
+~~~~~~~~~~~~~~~~~
+
+The text search assigns a numerical text score to indicate how relevant
+each result matches the search string. Often, you want to know which
+results are the most relevant.  
+
+To sort by relevance, you use the :doc:`Projections
+</fundamentals/builders/projections>`  and :doc:`Sorts
+</fundamentals/builders/sort>` builders.
```

Comment:
From the paragraphs below, I see you understand what the builders are for, but I think item 1 still hasn't been addressed.

When you start the sentence with "To sort by relevance", the remainder of the sentence should answer what operations must you perform to accomplish that. You don't need the Projects or Sorts builders to accomplish this.

Instead, let's try this:

1) Start the paragraph with "To perform a text search and sort the results by text score relevance, you must specify the score field as parameters in both your sort and projection methods."
This is describing how they need to perform collection.find().projection().sort() without mentioning the helpers.
2) Now, we talk about the helpers that you can use to specify the parameters to those methods. "You can use the Projection.metaTextScore() builder method to specify the projection and the Sorts.metaTextScore() builder method to specify the sort."
3) The above properly introduces the two new paragraphs you created to describe the builders.


If this doesn't make complete sense, let's discuss this over Zoom because it's an important distinction.